assets/scss/templates/_guidelines.scss

```diff
@@ -233,3 +233,467 @@ body {
   justify-content: center;
   align-items: center;
 }
+
+
+/*-----------------------------------------------------------------------------------------------*/
+/* New Layout */
+
+.wrapper-main {
+  display: flex;
+  flex-direction: column;
+  font-family: $dp-font-family-proximanova;
+}
+
+/* Intro Animatio Layers */
+.intro-open {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  pointer-events: none;
+  z-index: 10000;
+
+  .layer {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    transform-origin: top;
+    animation: introOpen .5s ease-in-out forwards;
+  }
+
+  .layer:nth-child(1) {
+    background: $dp-color-darkgrey;
+    z-index: 2;
+  }
+
+  .layer:nth-child(2) {
+    background: $dp-color-darkyellow;
+    z-index: 1;
+    animation-delay: .8s;
+  }
+
+}
+
+/* wrapper header*/
+
+.wrapper-header {
+  height: 170px;
+  display: flex;
+  position: relative;
+  background: $dp-color-white;
+
+  &:after {
+    position: absolute;
+    content: "";
+    height: 2%;
+    width: 90%;
+    left: 6%;
+    border-radius: 50%;
+    box-shadow: 0 0 5px rgba(0,0,0,0.2);
+    bottom: 1px;
+    z-index: -1;
+    opacity: 0;
+    animation: fadeInRight 0.5s linear forwards;
+    animation-delay: 2s;
+  }
+
+  .dp-logo {
+    width: 70px;
+    height: 70px;
+    margin: $dp-spaces--lv7 $dp-spaces--lv0 $dp-spaces--lv5 $dp-spaces--lv5;
+    display: inline-block;
+    opacity: 0;
+    animation: fadeInRight 0.5s linear forwards;
+    animation-delay: 4s;
+
+    img {
+      display: block;
+      width: 100%;
+    }
+
+  }
+
+}
+
+/* header text*/
+
+.banner-text {
+  display: block;
+  padding: $dp-spaces--lv7 $dp-spaces--lv0 $dp-spaces--lv5 $dp-spaces--lv3;
+
+    h2, h3 {
+    position: relative;
+    margin: $dp-spaces--lv0;
+    padding: $dp-spaces--lv0;
+    color: $dp-color-darkyellow;
+    line-height: 1.2em;
+    display: inline-block;
+    transform-origin: left;
+    text-transform: uppercase;
+    transform: scaleX(0);
+    animation: revealtextreverse 1s ease-in-out forwards;
+  }
+
+  h2 {
+    font-size: 2.5em;
+    animation-delay: 2s;
+    display: block;
+  }
+
+  h3 {
+    font-size: 1.5em;
+    animation-delay: 3s;
+  }
+
+  h2:before,
+  h3:before {
+    content: "";
+    position: absolute;
+    top: $dp-spaces--lv0;
+    left: $dp-spaces--lv0;
+    background: $dp-color-darkyellow;
+    width: 100%;
+    height: 100%;
+    transform-origin: right;
+    animation: revealtext 1s ease-in-out forwards;
+    animation-delay: 3s;
+  }
+
+  h3:before {
+    animation-delay: 4s;
+  }
+
+}
+
+.wrapper-container {
```

Comment:
Perhaps instead of wrapper for all this we could think of a more specialized name, what do you think about `guide`, `playground`, `styleguide`, I'm running out of imagination...